src/types/http.rs

```diff
@@ -25,13 +25,13 @@ impl Serialize for Action {
         match self {
             Action::Put(edn, None) => format!("[:crux.tx/put {}]", edn),
             Action::Put(edn, Some(date)) => format!(
-                "[:crux.tx/put {} #inst {}]",
+                "[:crux.tx/put {} #inst \"{}\"]",
```

Comment:
Why this is being added?

Reply:
because when we send a chrono time instance it sends just the date format, and Crux interprets it as a number. `#inst <date>` should contain a string with the date